src/core/ddsi/src/q_debmon.c

```diff
@@ -381,7 +381,7 @@ struct debug_monitor *new_debug_monitor (struct q_globals *gv, int port)
   if (ddsi_listener_listen (dm->servsock) < 0)
     goto err_listen;
   dm->stop = 0;
-  create_thread (&dm->servts, gv, "debmon", debmon_main, dm);
+  (void) create_thread (&dm->servts, gv, "debmon", debmon_main, dm);
```

Comment:
No way. It is only an optional monitoring thing, but even so, it ought to check whether thread creation succeeded and gracefully handle failure, and certainly mustn't hide _not_ checking!

Reply:
I will add a check on the return code of create_thread.